src/mlpack/methods/ann/layer/max_pooling.hpp

```diff
@@ -159,7 +159,7 @@ class MaxPooling
   bool Deterministic() const { return deterministic; }
   //! Modify the value of the deterministic parameter.
   bool& Deterministic() { return deterministic; }
-  
+ 
```

Comment:
@Aakash-kaushik , have a look here. I removed that line with extra space and added a new blank line. Now, I am unable to understand why is it showing two errors here - one is "Weird number of spaces at line-start" and the other is "Line ends in whitespace".